src/ai/backend/storage/filebrowser/filebrowser.py

```diff
@@ -0,0 +1,208 @@
+    service_port = get_available_port(port_range)
+    running_docker_containers = await get_filebrowsers()
+    if len(running_docker_containers) >= max_containers:
+        print(
+            "Can't create new container. Number of containers exceed the maximum limit.",
+        )
+        return ("0", 0, "0")
+    await prepare_filebrowser_app_config(settings_path, service_port)
+    async with closing_async(aiodocker.Docker()) as docker:
+        config = {
+            "Cmd": [
+                "/filebrowser_dir/start.sh",
+                f"{user_id}",
+                f"{group_id}",
+                f"{service_port}",
+            ],
+            "ExposedPorts": {
+                f"{service_port}/tcp": {},
+            },
+            "Image": image,
+            "HostConfig": {
+                "PortBindings": {
+                    f"{service_port}/tcp": [
+                        {
+                            "HostIp": f"{service_ip}",
+                            "HostPort": f"{service_port}/tcp",
+                        },
+                    ],
+                },
+                "CpuCount": cpu_count,
+                "Memory": memory,
+                "Mounts": [
+                    {
+                        "Target": "/filebrowser_dir/",
+                        "Source": f"{settings_path}",
+                        "Type": "bind",
+                    },
+                ],
+            },
+        }
+        for vfolder in vfolders:
+            filebrowser_mount_path = str(
+                volume_obj.mangle_vfpath(UUID(vfolder["vfid"])),
+            )
+            config["HostConfig"]["Mounts"].append(
+                {
+                    "Target": f"/data/{str(vfolder['name'])}",
+                    "Source": filebrowser_mount_path,
+                    "Type": "bind",
+                },
+            )
+        container_name = f"ai.backend.container-filebrowser-{service_port}"
+        container = await docker.containers.create_or_replace(
+            config=config,
+            name=container_name,
+        )
+        container_id = container._id
+        await container.start()
+    tracker_db = FilebrowserTrackerDB(db_path)
+    await tracker_db.insert_new_container(
+        container_id,
+        container_name,
+        service_ip,
+        service_port,
+        config,
+        "RUNNING",
+        str(datetime.now().strftime("%Y-%m-%d %H:%M:%S")),
+    )
+    return service_ip, service_port, container_id
+
+
+async def recreate_container(container_name: str, config: dict[str, Any]) -> None:
+    async with closing_async(aiodocker.Docker()) as docker:
+        try:
+            docker = aiodocker.Docker()
+            container = await docker.containers.create_or_replace(
+                config=config,
+                name=container_name,
+            )
+            await container.start()
+        except Exception as e:
+            print("Failure to recreate container ", e)
```

Comment:
We need to notify user that the operation is failed, instead of just eating up raised Exception.